scripts/Muon/GUI/FrequencyDomainAnalysis/frequency_context.py

```diff
@@ -22,10 +22,15 @@ class FFT(object):
 
     def __init__(self, ws_freq_name, Re_run, Re, Im_run, Im, phasequad):
         self.Re_run = Re_run
-        self.Im_run = Im_run
+
         self.ws_freq_name = ws_freq_name
         self.Re = Re
-        self.Im = Im
+        if Im == "" and Im_run == "":
```

Comment:
What happens if one of these is an empty string and not the other or are they always tied together?

Reply:
should be tied together:

Im run is set https://github.com/mantidproject/mantid/blob/master/scripts/Muon/GUI/FrequencyDomainAnalysis/FFT/fft_presenter_new.py#L214

Im group/pair is from 
https://github.com/mantidproject/mantid/blob/master/scripts/Muon/GUI/FrequencyDomainAnalysis/FFT/fft_presenter_new.py#L221
which calls
https://github.com/mantidproject/mantid/blob/master/scripts/Muon/GUI/Common/ADSHandler/workspace_naming.py#L52

so both should return empty strings